app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+# SessionsController
+class SessionsController < ApplicationController
+  def new; end
+
+  def create
```

Comment:
You need error handling here supposing makes a mistake signing in or if a user tries to press login without placing a username